mongo-dao/src/main/java/com/kovan/service/NewsRepositoryService.java

```diff
@@ -0,0 +1,41 @@
+package com.kovan.service;
+
+import com.kovan.dto.NewsDto;
+import com.kovan.mapper.NewsMapper;
+import com.kovan.entity.NewsEntity;
+import com.kovan.repository.NewsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+@Service
+public class NewsRepositoryService {
+
+    @Autowired
+    private NewsRepository newsRepository;
+
+    @Autowired
+    private NewsMapper newsMapper;
+
+    public NewsDto saveNewsInDb(NewsDto newsDto) {
+        if (Objects.isNull(newsDto)) {
+            throw new IllegalArgumentException("NewsDto cannot be null");
```

Comment:
check using custom exception

Reply:
Used custom exception for exception handling.